freqtrade/rpc/telegram.py

```diff
@@ -215,8 +215,8 @@ def _status(self, update: Update, context: CallbackContext) -> None:
                     "*Open Rate:* `{open_rate:.8f}`",
                     "*Close Rate:* `{close_rate}`" if r['close_rate'] else "",
                     "*Current Rate:* `{current_rate:.8f}`",
-                    "*Close Profit:* `{close_profit}`" if r['close_profit'] else "",
-                    "*Current Profit:* `{current_profit:.2f}%`",
+                    "*Close Profit:* `{close_profit_pct}`" if r['close_profit_pct'] else "",
```

Comment:
related to the comment above: "" should be printed for close_profit_pct == None, not for 0.0